content/applications/finance/payment_acquirers.rst

```diff
@@ -292,6 +292,16 @@ Payment journal
 
 The **Payment journal** selected for your payment acquirer must be a *Bank* journal.
 
+.. _payment_acquirers/published_unpublished:
+
+Published/Unpublished Flag
+**************************
+
+This button on the top right corner of your acquirer lets you control visibility on the customer's
```

Comment:
```suggestion
This button in the top right corner of your acquirer lets you control visibility on the customer's
```
I could not find that button on a master runbot, so without context, I guess "in" is a better choice of words.